### ISSUE TITLE
Clicking topic list does not change filter and should probably not use the default page type

### ISSUE THREAD

**Issue by mikecreuzer** [Audentio/xf2addon-issues#657]
Two issues here, very similar:

1. Going to Topic List > clicking an item, will not unset the filter I have until I refresh
2. It takes me to the default, in my case trending. But really this should take me to a view sorted by most recent (I think). In other words, the topic list tells me how many messages/threads (in my case 4 and 20 bro) but it only shows me one thread as thats all thats trending. It might be confusing. 

Simply put: Do we have a view that shows all thread by topic? Not newest, not trending, etc. Just all posts, like what a node would do. I think thats what we need here

**Comment by LukasWieditz** [comment]
That view is literally what we deleted before because we didn't want it...

**Comment by mikecreuzer** [comment]
Hmm it may be needed for the simple fact that 

A) things won't be new or trending forever

B) people will want to just see all threads all the way back to start of board. Our routes are limited in results

**Comment by LukasWieditz** [comment]
We agreed in earlier discussions, that 20 (I think it's 20, can't recall from the top of my head) pages max will suffice and that people that want earlier threads will use the search, hence why we deleted the topic specific routes and all functionality associated with them (in addition to the fact that someone termed them as "navigation nightmare").
